Text/TextQualityWatchdog/Util/NormalizeData.py

```diff
@@ -0,0 +1,100 @@
+###
+# This script is used to preprocess text files located in a source directory
+# and save the cleaned text files in a destination directory as a tsv
+###
+
+import os
+import re
+import pickle
+import pandas as pd
+
+APP_ROOT = os.path.dirname(os.path.abspath(__file__))
+APP_ROOT_STR = APP_ROOT.__str__()
+
+# Source directory containing all of the text files extracted from tika
+SOURCE_DIR = 'ExtractedTextData'
+
+# Destination directory to save the final tsv file to
+DEST_DIR = 'Data'
+
+# Metadata directory that we saved the tika-eval metrics to in a pickle file
+METADATA_DIR = 'Metadata'
+
+def _simplify_punctuation(text):
+    """
+    This function simplifies doubled or more complex punctuation. The exception is '...'.
+    """
+    corrected = str(text)
+    corrected = re.sub(r'([!?,;])\1+', r'\1', corrected)
+    corrected = re.sub(r'\.{2,}', r'...', corrected)
+    return corrected
+
+def _normalize_whitespace(text):
+    """
+    This function normalizes whitespaces, removing duplicates.
+    """
+    corrected = str(text)
+    corrected = re.sub('\s', ' ', corrected)
+    corrected = re.sub(r"( )\1+",r"\1", corrected)
+    return corrected.strip(" ")
```

Comment:
Just inmport PreProcessData and reuse these methods from there (or visa versa). Best not to dupe code if at all possible

Reply:
Done! Removed redundant methods and added import statement.